### PR TITLE
Add tracking code for post author, post id.

### PR DESCRIPTION
**Pull request by fumikito**
- Do refactoring
- Change namespace.
- Add new settings for custom definition.

If you enable [WooCommerce Google Analytics Integration](https://ja.wordpress.org/plugins/woocommerce-google-analytics-integration/), codes abover will be displayed.

```js
ga('set', 'dimension1', 3); // Means set custom dimention slot 1 as 3.
```

### CODE REVIEW

**Comment by ko31** [comment]
@fumikito This is a great feature!

Is this code unnecessary?
https://github.com/tarosky/google-analytics-reports/blob/feature/keyword-search/src/Tarosky/GoogleAnalyticsReports/Tracker.php#L32

File: src/Tarosky/GoogleAnalyticsReports/Tracker.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	 * Get extra tags.
+	 */
+	public function get_extra_tags() {
+		$codes = [];
+		foreach ( [
+			'post_id' => is_singular() ? get_queried_object_id() : null,
+			'author'  => is_singular() ? get_queried_object()->post_author : null,
+		] as $key => $value ) {
+			$index = get_option( "google-analytics-reports-{$key}" );
+			var_dump( $index, $value, $key );
```
Comment (fumikito) [inline src/Tarosky/GoogleAnalyticsReports/Tracker.php]:
```suggestion
```

This line is unnecessary.

Comment (fumikito) [reply src/Tarosky/GoogleAnalyticsReports/Tracker.php]: @ko31 Please review this comment.

**Review by fumikito — COMMENTED** [review]
@ko31 Removed var_dump.

File: src/Tarosky/GoogleAnalyticsReports/Tracker.php (hunk 1 above)
Comment (ko31) [reply src/Tarosky/GoogleAnalyticsReports/Tracker.php]: @fumikito thanks!!

**Pull request merged**